ops/storage-ddls/cassandra-setup.cql

```diff
@@ -78,88 +76,126 @@ CREATE TYPE features (
     entities frozen<set<computedentities>>
 );
 
-CREATE TABLE computedtiles (
-    tilex int,
-    tiley int,
-    tilez int,
-    tileid text,
+DROP TABLE IF EXISTS conjunctivetopics;
+CREATE TABLE conjunctivetopics (
+    topic text,
+    conjunctivetopic text,
+    mentions int,
+    PRIMARY KEY(topic, mentions, conjunctivetopic)
+) WITH CLUSTERING ORDER BY(mentions DESC);
+
+DROP TABLE IF EXISTS populartopics;
+CREATE TABLE populartopics (
     periodstartdate timestamp,
     periodenddate timestamp,
     periodtype text,
+    pipelinekey text,
     period text,
-    pipeline text,
-    sourceid text,
+    tilez int,
+    tilex int,
+    tiley int,
+    externalsourceid text,
     topic text,
-    topiclangcode text,
+    mentionCount int,
     insertion_time timestamp,
-    heatmap text,
-    placeids frozen<set<text>>,
-    computedfeatures frozen<features>,
-    PRIMARY KEY ((tilex, tiley, tilez), periodstartdate, periodenddate, pipeline, sourceid, topic)
+    PRIMARY KEY ((periodtype, pipelinekey, externalsourceid, tilez, topic, period), tilex, tiley, periodstartdate, periodenddate)
 );
 
-CREATE INDEX ON computedtiles (tileid);
-CREATE INDEX ON computedtiles (period);
-CREATE INDEX ON computedtiles (computedfeatures);
-CREATE INDEX ON computedtiles (periodtype);
-
-CREATE TABLE computedplaces (
-    placeid text,
-    placename text,
+DROP TABLE IF EXISTS computedtiles;
+CREATE TABLE computedtiles (
     periodstartdate timestamp,
     periodenddate timestamp,
     periodtype text,
+    pipelinekey text,
     period text,
-    pipeline text,
-    sourceid text,
-    topic text,
-    topiclangcode text,
+    tilez int,
+    tilex int,
+    tiley int,
+    externalsourceid text,
+    mentioncount int,
+    avgsentiment int,
+    heatmap text,
+    placeids frozen<set<text>>,
     insertion_time timestamp,
-    computedfeatures frozen<features>,
-    PRIMARY KEY ((placeid), periodstartdate, periodenddate, pipeline, sourceid, topic)
+    conjunctiontopics tuple<text, text, text>,
+    PRIMARY KEY ((periodtype, conjunctiontopics, tilez, period), tilex, tiley, periodstartdate, periodenddate, pipelinekey, externalsourceid)
```

Comment:
Any chance we can remove `periodstartdate` and `periodenddate` from the PK since they are already covered by the `period` column?